setup.sh

```diff
@@ -5,15 +5,32 @@
 
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 CORENLP_VERSION='4.2.0'
-SPRINGER_API_KEY='aa526f60fdf49130aa089a4728046233'
```

Comment:
Are you ok with leaving this API key in the history of the repo?

Reply:
yes, good point.... i need to purge it.....

Reply:
If it is too hard to do that, maybe you can just disable it and generate another API for you?

Reply:
yeah, I think that's a better option.... looking into it

Reply:
ok API key has been regenerated